app/assets/stylesheets/welcome.css.scss

```diff
@@ -1,3 +1,6 @@
 // Place all the styles related to the welcome controller here.
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
+.jumbotron{
+  margin-top: 20px !important;
```

Comment:
Try to avoid using `!important` in CSS, there's usually another way to solve these styling issues without `!important`.